CMakeLists.txt

```diff
@@ -16,7 +16,6 @@ include(Common)
 
 set(COMMON_PROJECT_DOMAIN eu.humanbrainproject)
 set(ZEQ_DESCRIPTION "ZeroEQ - Zero Event Queue")
-set(ZEQ_DEPENDENT_LIBRARIES Lunchbox)
```

Comment:
That's right. The prototype with out Lunchbox at all didn't have it, but now the URI is leaked.